### ISSUE TITLE
Guilds of Ravnica Guild Kits

### ISSUE THREAD

**Issue by fenhl** [taw/magic-search-engine#123]
This set is on Gatherer and on MTG JSON v4 but not on v3.

**Comment by taw** [comment]
Right, v4 transition is going to complicate it a bit. Fortunately I can import single sets from v4 or Gatherer even if not all v4 issues are resolved yet.

**Comment by taw** [comment]
So in v4, `releaseDate` is missing, [[Burn // Turn]] is broken like every other split card in v4, and all the text and rulings differences need to be checked manually since I have limited trust in it.

But I should be able to get something working based on that.

**Comment by taw** [comment]
I got it added, but that required a lot of manually messing with GK1, there's still a lot of v4 issues and I'm not too happy about it.
